### PR TITLE
IDVA3-1368 Rle verified screen

### PR DESCRIPTION
**Pull request by JasSmith-ch**
- Added static content for the RLE Verified screen

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: locales/en/rle-verified.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "rle_verified_title": "RLE confirmed",
```
Comment (eedwards0) [inline locales/en/rle-verified.json:2]: Please change to 'PSC confirmed'

Comment (eedwards0) [inline locales/en/rle-verified.json:2]: In the prototype v5, the title is 'PSC confirmed'. I guess we better match this one :).

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: locales/en/rle-verified.json
Comment (eedwards0) [inline locales/en/rle-verified.json:1]: Page title is 'PSC confirmed' in prototype v5.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by eedwards0 — CHANGES_REQUESTED** [review]
Please see individual comments

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by eedwards0 — APPROVED** [review]
All now working as expected in Docker

**Pull request merged**